client/state/selectors/is-calypsoify-gutenberg-enabled.js

```diff
@@ -5,17 +5,14 @@
 import { isEnabled } from 'config';
 import isVipSite from 'state/selectors/is-vip-site';
 import { isJetpackSite } from 'state/sites/selectors';
+import { abtest } from 'lib/abtest';
 
-export const isCalypsoifyGutenbergEnabled = ( state, siteId ) => {
-	if ( ! siteId ) {
-		return false;
-	}
-
-	const calypsoifyGutenberg = isEnabled( 'calypsoify/gutenberg' );
-	const isJetpack = isJetpackSite( state, siteId );
-	const isVip = isVipSite( state, siteId );
-
-	return calypsoifyGutenberg && ! isJetpack && ! isVip;
-};
+export const isCalypsoifyGutenbergEnabled = ( state, siteId ) =>
+	siteId
+		? isEnabled( 'gutenberg/opt-in' ) &&
```

Comment:
This should be `isEnabled( 'calypsoify/gutenberg' )`

Reply:
I remove that flag altogether. `gutenberg/opt-in` now controls both the `calypsoify` and `opt-in` because it doesn't make sense to have these separated (provided that we disregard the existence of `gutenberg` flag).